terraform/cloudfoundry/dns-records.tf

```diff
@@ -38,6 +22,14 @@ resource "aws_route53_record" "cf_ssh_proxy" {
   records = ["${aws_elb.ssh_proxy.dns_name}"]
 }
 
+resource "aws_route53_record" "system_domain" {
+  zone_id = "${var.system_dns_zone_id}"
+  name    = "${var.system_dns_zone_name}"
+  type    = "CNAME"
```

Comment:
You can't put a `CNAME` at the root of a zone. You'll have to use an `ALIAS` here. See `cf_apps_apex` below for an example.

We should probably also call this `system_apex` for consistency.

Reply:
Good spot. Done.